pkg/quota/v1/evaluator/core/pods_test.go

```diff
@@ -438,9 +439,46 @@ func TestPodEvaluatorUsage(t *testing.T) {
 				generic.ObjectCountQuotaResourceNameFor(schema.GroupResource{Resource: "pods"}): resource.MustParse("1"),
 			},
 		},
+		"pod overhead": {
+			pod: &api.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					DeletionTimestamp:          &deletionTimestampNotPastGracePeriod,
+					DeletionGracePeriodSeconds: &terminationGracePeriodSeconds,
+				},
+				Status: api.PodStatus{
+					Reason: node.NodeUnreachablePodReason,
+				},
+				Spec: api.PodSpec{
+					Overhead: api.ResourceList{
+						api.ResourceCPU: resource.MustParse("1"),
+					},
+					Containers: []api.Container{
+						{
+							Resources: api.ResourceRequirements{
+								Requests: api.ResourceList{
+									api.ResourceCPU: resource.MustParse("1"),
+								},
+								Limits: api.ResourceList{
+									api.ResourceCPU: resource.MustParse("2"),
+								},
+							},
+						},
+					},
+				},
+			},
+			usage: corev1.ResourceList{
```

Comment:
this test case tests for usage, not for quota, correct? Sorry I might be missing something

Reply:
According to [the KEP](https://github.com/kubernetes/enhancements/blob/7e69f3be9783e33f597b02de8c952b6eca1e8273/keps/sig-node/688-pod-overhead/README.md#resourcequota-changes) (`The controller will be updated to add the pod Overhead to the container resource request summation.`), I would say that `the container resource request summation` means the usage. And #99577  requested the same thing.

Yes, the test is for usage, and resourcequota controller ensures the usage to not exceed the quota